tests/dummy/app/application/template.hbs

```diff
@@ -1,3 +1,3 @@
-{{#nrg-application title="NRG UI"}}
+{{#nrg-application title="NRG UI" clicked=(action "clickedLink")}}
```

Comment:
Since this action is no longer required to be implemented and you aren't doing anything with it in the dummy app, please remove it.